code-challenges.test.js

```diff
@@ -13,55 +15,122 @@
+// same error occured but then I revise syntax.
+// originally i put array.filter but it wasnt counted as a function according to error so I tried const name and attached to filter
+
+describe("returnLarger", () => {
+  it("takes in a number then returns an array.", () => {
+    const fibLength1 = 6
+    const fibLength2 = 10
+
+    // Expected output: [1, 1, 2, 3, 5, 8]
+    // Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
+    expect(returnLarger(fibLength1)).toEqual[1, 1, 2, 3, 5, 8]
+    expect(returnLarger(fibLength2)).toEqual[1, 1, 2, 3, 5, 8, 13, 21, 32, 34, 55]
+  })
+})
+// let  returnLarger2 = (arr, num) => fibLength2.filter(fibLength2 => 10 > num); {
+// } this is from past attempt 
 
-const fibLength1 = 6
-// Expected output: [1, 1, 2, 3, 5, 8]
-
-const fibLength2 = 10
-// Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
 
 // b) Create the function that makes the test pass.
+const returnLarger = (num) => {
+  let emptyArr = [1,1]
+  for (i = 2; i < num; i++) {
+    emptyArr.push(emptyArr[i-2] + emptyArr[i-1])
+  }
+  return emptyArr
+  }
+  
+
 
 // --------------------2) Create a function that takes in an object and returns an array of the numbers sorted from least to greatest.
 // Hint: Check out this resource: Object.values() https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_objects/Object/values
 
 // a) Create a test with expect statements for each of the variables provided.
+// focus on making function that sorts the objects value from least to greatest
+// first make function and name it with self documenting name
+// use console log that spits out output as if it was manually entered first
+// put expect statements under objects to expect the function and also the expected outcome listed
+
+describe("days", () => {
+  it("sorts object values least to greatest.", () => {
+
+    const studyMinutesWeek1 = {
+      sunday: 90,
+      monday: 30,
+      tuesday: 20,
+      wednesday: 15,
+      thursday: 30,
+      friday: 15,
+      saturday: 60
+    }
+
+    // Expected output: [15, 15, 20, 30, 30, 60, 90]
+
+    const studyMinutesWeek2 = {
+      sunday: 100,
+      monday: 10,
+      tuesday: 45,
+      wednesday: 60,
+      thursday: 20,
+      friday: 15,
+      saturday: 65
+    }
+    expect(leastToGreatest(studyMinutesWeek1)).toEqual[15, 15, 20, 30, 30, 60, 90]
+    expect(leastToGreatest(studyMinutesWeek2)).toEqual[10, 15, 20, 45, 60, 65]
+  })
+})
+
+
 
-const studyMinutesWeek1 = {
-  sunday: 90,
-  monday: 30,
-  tuesday: 20,
-  wednesday: 15,
-  thursday: 30,
-  friday: 15,
-  saturday: 60
-}
-// Expected output: [15, 15, 20, 30, 30, 60, 90]
-
-const studyMinutesWeek2 = {
-  sunday: 100,
-  monday: 10,
-  tuesday: 45,
-  wednesday: 60,
-  thursday: 20,
-  friday: 15,
-  saturday: 65
-}
 // Expected output: [10, 15, 20, 45, 60, 65, 100]
 
 // b) Create the function that makes the test pass.
+let leastToGreatest = (object)  => {
+  return object.sort((a,b ) => a - b);
+  
+  
```

Comment:
Please include your pseudocode for this question.  Also, we would like to see your terminal output.  Just a note - you can't call sort on an object.